### PR TITLE
Fix Issue #294

### PR DESCRIPTION
**Pull request by VikingsFan**
Change  \<see langword="..."/>  to ...
              \<see>...\</see> to ...
Also fix a build bug.

### CODE REVIEW

File: OData/WebApiOData.msbuild [hunk 1]
```diff
@@ -17,12 +17,6 @@
     <SkipStrongNamesXml>$(MSBuildThisFileDirectory)tools\SkipStrongNames.xml</SkipStrongNamesXml>
     <NuGetExeFile>$(MSBuildThisFileDirectory).nuget\NuGet.exe</NuGetExeFile>
   </PropertyGroup>
-
-  <UsingTask AssemblyFile=".\packages\xunit.1.9.1\lib\net20\xunit.runner.msbuild.dll" TaskName="Xunit.Runner.MSBuild.xunit"/>
```
Comment (xuzhg) [inline OData/WebApiOData.msbuild]: why remove these lines in this fix?

File: OData/WebApiOData.msbuild [hunk 2]
```diff
@@ -17,12 +17,6 @@
...
-  
-  <ItemGroup>
-    <TestDLLsXunit Include="bin\$(Configuration)\FunctionalTest\*.Test.dll;bin\$(Configuration)\FunctionalTest\*.Test.*.dll;bin\$(Configuration)\FunctionalTest\NetCore\*.Test.dll" />
```
Comment (xuzhg) [inline OData/WebApiOData.msbuild]: And this?

File: OData/src/System.Web.Http.OData/OData/Builder/Conventions/Attributes/AttributeConvention.cs [hunk 3]
```diff
@@ -16,7 +16,7 @@ internal abstract class AttributeConvention : IConvention
         /// Initializes a new instance of the <see cref="AttributeConvention"/> class.
         /// </summary>
         /// <param name="attributeFilter">A function to test whether this convention applies to an attribute or not.</param>
-        /// <param name="allowMultiple"><see langword="true"/> if the convention allows multiple attribues; otherwise, <see langword="false"/>.</param>
+        /// <param name="allowMultiple">true if the convention allows multiple attribues; otherwise, false.</param>
```
Comment (xuzhg) [inline OData/src/System.Web.Http.OData/OData/Builder/Conventions/Attributes/AttributeConvention.cs]: <c>true</c>

Comment (xuzhg) [inline OData/src/System.Web.Http.OData/OData/Builder/Conventions/Attributes/AttributeConvention.cs]: same for others

File: OData/WebApiOData.msbuild (hunk 1 above)
Comment (VikingsFan) [inline OData/WebApiOData.msbuild]: no change, just revert this file to fix a tfs build error

**Comment by xuzhg** [comment]
sign-off.

**Comment by xuzhg** [comment]
Merged at: eeabd634eef7e422049930681c88859e658c2608

**Comment by congysu** [comment]
The issue #294, and related PR #332.

**Pull request closed without merging**